xml/troubleshooting.xml

```diff
@@ -685,12 +685,12 @@
   </sect2>
   
   <sect2 xml:id="sec-trouble-boot-dual-boot">
-   <title> <productname>GRUB 2</productname> fails during reboot on a dual-boot system</title>
+   <title><productname>GRUB 2</productname> fails during reboot on a dual-boot
```

Comment:
It may be better to use an entity here:

```suggestion
   <title><productname>&grub;</productname> fails during reboot on a dual-boot
```

(which is resolved as "GRUB 2" according to its definition in `xml/entity-decl.ent`)

Reply:
changed as suggested